### PR TITLE
WWW Mobile/Safari fixes - AKA Safari likes being difficult

### PR DESCRIPTION
**Pull request by krishnaglick**
Safari has odd opinions. Hiding images from HowGrouparooWorks at smaller screen sizes

## Change description

Description here

### Development

- [x] Application changes have been tested appropriately

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: components/home/actionable/Actionable.tsx [hunk 1]
```diff
@@ -22,13 +22,15 @@ export const Actionable = () => {
...
-                </Button>
+                <div>
+                  <Button
+                    type="button"
+                    variant="secondary"
+                    className="rounded-pill"
+                  >
+                    Learn More
+                  </Button>
+                </div>
```
Comment (krishnaglick) [inline components/home/actionable/Actionable.tsx:33]: Safari

File: components/home/features/Features.tsx [hunk 2]
```diff
@@ -42,23 +42,23 @@ export const Features = () => {
...
+        <Col md={4} xs={12}>
           <Sliders />
           <h4>Conventions</h4>
           <p>
             Maintainable conventions so everyone on the team can easily
             understand and modify Grouparoo.
           </p>
         </Col>
-        <Col>
+        <Col md={4} xs={12}>
```
Comment (krishnaglick) [inline components/home/features/Features.tsx:61]: Mobile

File: components/home/how-it-works/HowGrouparooWorks.module.scss [hunk 3]
```diff
@@ -3,9 +3,9 @@
     display: flex;
     flex-direction: row;
 
-    // BS Medi
+    // BS Med
     @media screen and (max-width: 768px) {
-      flex-direction: column;
+      display: none;
```
Comment (krishnaglick) [inline components/home/how-it-works/HowGrouparooWorks.module.scss:8]: Mobile

File: components/home/how-it-works/HowGrouparooWorks.tsx [hunk 4]
```diff
@@ -6,7 +6,7 @@ export const HowGrouparooWorks = () => {
   return (
     <Container className={`${styles.container} py-3`}>
       <Row>
-        <Col>
+        <Col md={6} xs={12}>
```
Comment (krishnaglick) [inline components/home/how-it-works/HowGrouparooWorks.tsx:9]: Mobile

File: components/home/partners/logos/MindfulChef.tsx [hunk 5]
```diff
@@ -4,6 +4,7 @@ export const MindfulChef = ({ className }: { className?: string }) => (
     preserveAspectRatio="xMidYMid meet"
     shapeRendering="geometricPrecision"
     xmlns="http://www.w3.org/2000/svg"
+    width="1000px"
```
Comment (krishnaglick) [inline components/home/partners/logos/MindfulChef.tsx:7]: Safari

File: components/home/partners/logos/OneMedical.tsx [hunk 6]
```diff
@@ -1,5 +1,10 @@
 export const OneMedical = () => (
-  <svg viewBox="0 0 204 35" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <svg
+    viewBox="0 0 204 35"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    width="1000px"
```
Comment (krishnaglick) [inline components/home/partners/logos/OneMedical.tsx:6]: Safari

File: components/home/partners/logos/Raisedev.tsx [hunk 7]
```diff
@@ -1,5 +1,10 @@
 export const Raisedev = () => (
-  <svg viewBox="0 0 114 22" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <svg
+    viewBox="0 0 114 22"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    width="1000px"
```
Comment (krishnaglick) [inline components/home/partners/logos/Raisedev.tsx:6]: Safari

Comment (evantahler) [reply components/home/partners/logos/Raisedev.tsx:6]: How did you figure these dimensions out?  That's cool.

**Review by evantahler — APPROVED** [review]
Things show up on safari now!

File: components/home/partners/logos/Raisedev.tsx (hunk 7 above)
Comment (krishnaglick) [reply components/home/partners/logos/Raisedev.tsx:6]: Oh, a max-width is set in the CSS so I just tacked a number of so if we ever increase the css we're unlikely to run into an issue with clashing width/max-width.

**Comment by teallarson** [comment]
image in safari on mobile! Other spots look nice and clean too.

**Review by teallarson — APPROVED** [review]
nice and clean

**Pull request merged**